server/lib/handlebars.js

```diff
@@ -108,7 +108,11 @@ handlebars.registerHelper('moment', (value, props) => {
 });
 
 handlebars.registerHelper('moment-timezone', value => {
-  return moment().tz(value).format('Z');
+  if (!value) {
+    return '';
+  } else {
+    return moment().tz(value).format('Z');
+  }
```

Comment:
An error popped up during a test here, where we had an event with `timeZone=null`. I've added the timezone to the event, but it seems helpful to not crash in this case anyway.